plugin.video.vrt.nu/resources/language/resource.language.en_gb/strings.po

```diff
@@ -68,9 +71,21 @@ msgctxt "#32092"
 msgid "Categories"
 msgstr "Categories"
 
+msgctxt "#32093"
+msgid "Most recent"
+msgstr "Most recent"
+
+msgctxt "#32094"
+msgid "Season"
+msgstr "Season"
+
+msgctxt "#32095"
+msgid "Episode"
+msgstr "Episode"
+
 msgctxt "#32100"
-msgid "Livestreams"
-msgstr "Livestreams"
+msgid "Watch live"
```

Comment:
I don't like imperative menu-items/titles, and since the other menu-items aren't imperative, I would rather propose to name this "Live TV" or "Live channels" if "Live streams" is considered too "techy".

Reply:
I agree, we should not give orders to our users. They freely choose what to do with the addon ;-)

Reply:
Could we maybe use "Live TV streams" instead ? I think it's the best of both worlds, it includes "streams" so there's no mixup that this would be antenna/cable and it's clear it's about "Live TV".

I know I am splitting hairs here, bear with me ;-)